src/libraries/System.Diagnostics.DiagnosticSource/src/System/Diagnostics/Metrics/MetricsEventSource.cs

```diff
@@ -414,7 +414,7 @@ private string FormatTags(KeyValuePair<string, string>[] labels)
                 StringBuilder sb = new StringBuilder();
                 for (int i = 0; i < labels.Length; i++)
                 {
-                    sb.AppendFormat(CultureInfo.InvariantCulture, "{0}={1}", labels[i].Key, labels[i].Value);
+                    sb.Append(CultureInfo.InvariantCulture, $"{labels[i].Key}={labels[i].Value}");
```

Comment:
This library compiles for more than just netcoreappcurrent, and this overload doesn't exist earlier than .NET 6.  Please revert all of the changes to this library.

Reply:
How about I add an `#if NET_6_0_OR_GREATER`?

Reply:
Can you share the benchmark that indicates it's worth the added complexity and maintenance burden?

Reply:
OK, never mind, I will revert it.